engines/python/setup/djl_python/deepspeed.py

```
@@ -155,12 +182,35 @@ def _parse_properties(self, properties):
                 "trust_remote_code").lower() == "true"
         if "revision" in properties:
             self.revision = properties['revision']
+
+        # SmoothQuant properties
+        self._parse_smoothquant_properties(properties)
+
         if properties.get("deepspeed_config_path"):
             with open(properties.get("deepspeed_config_path"), "r") as f:
                 self.ds_config = json.load(f)
         else:
             self.ds_config = self._get_ds_config(properties)
 
+    def _parse_smoothquant_properties(self, properties):
+        if 'quantize' in properties:
+            if properties['quantize'] not in SUPPORTED_QUANTIZATION_MODE:
+                self.logger.warning(
+                    f"DeepSpeed does not currently support quantization mode: ${properties['quantize']}, "
+                    f"this setting will be ignored.")
+                return
+            self.quantize_mode = properties['quantize']
+        if 'smoothquant_alpha' in properties:
+            try:
+                if 0 <= float(properties['smoothquant_alpha']) <= 1:
+                    self.smoothquant_alpha = properties['smoothquant_alpha']
+                else:
+                    self.logger.warning(f"${properties['smoothquant_alpha']} is not between 0 and 1, "
```

We should throw an exception here?